sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -160,3 +161,23 @@ export function toBuffer(input: any): Buffer {
   return result;
 }
 
+/**
+ * Helper function to retrieve active receiver name, if it exists.
+ *
+ */
+export function getAssociatedReceiverName(
+  clientEntityContext: ClientEntityContext,
+  sessionId?: string
+): string {
+  let receiverName: string;
+  if (sessionId !== undefined) {
+    if (clientEntityContext.messageSessions[sessionId]) {
+      receiverName = clientEntityContext.messageSessions[sessionId].name;
+    }
+  } else if (clientEntityContext.batchingReceiver) {
+    receiverName = clientEntityContext.batchingReceiver.name;
+  } else if (clientEntityContext.streamingReceiver) {
+    receiverName = clientEntityContext.streamingReceiver.name;
+  }
+  return receiverName!;
```

Comment:
Since the return value can be `undefined` as well, please update the return type to be `string | undefined` and remove the `!` here

We use `!` on a variable whose type allows `undefined` but we are sure that the value is not undefined. It is a way to tell typescript compiler to stop complaining as we know the variable cannot be undefined, but couldn't change the type, maybe because it was declared in a code section we don't own.

In this case, we define the variable `receiverName` and we know it can have `undefined`, so `!` is misplaced here